frontend/src/shared/components/IconBadgeList/IconBadgeList.tsx

```diff
@@ -0,0 +1,26 @@
+import { PLATFORM_ICON_MAP, TECH_ICON_MAP } from "./constants/iconMap";
+import IconBadge from "./IconBadge/IconBadge";
+import * as S from "./IconBadgeList.styled";
+
+interface IconBadgeListProps {
+  iconBadges: string[];
+}
```

Comment:
```suggestion
interface IconBadgeListProps {
  iconBadges: 기술 스택 + 플랫폼들의 키 타입[];
}
```

여기도 타입이 잘 잡혀야할 것 같네요 아래에 기술 스택, 플랫폼들을 합친 `iconBadgeMap` 객체의 키 값으로 타입을 사용하면 될 것 같아요🙃

그리고 `./constants/iconMap` 이라는 폴더에서 상수를 정의해주신 것 같은데 `iconBadgeMap` 여기에 두셨어야 하는 이유가 있으셨을까요??